modules/mono/glue/GodotSharp/GodotSharp/Core/Vector2.cs

```diff
@@ -512,24 +512,24 @@ public Vector2 Sign()
         /// Returns the result of the spherical linear interpolation between
         /// this vector and <paramref name="to"/> by amount <paramref name="weight"/>.
         ///
-        /// Note: Both vectors must be normalized.
+        /// This method also handles interpolating the lengths if the input vectors have different lengths.
+        /// For the special case of one or both input vectors having zero length, this method behaves like [method lerp].
```

Comment:
`[method lerp]` is not valid syntax for the C# XML documentation comments.

```suggestion
        /// This method also handles interpolating the lengths if the input vectors
        /// have different lengths. For the special case of one or both input vectors
        /// having zero length, this method behaves like <see cref="Lerp"/>.
```